src/main/java/org/folio/invoices/utils/ErrorCodes.java

```diff
@@ -12,7 +12,7 @@
   PO_LINE_UPDATE_FAILURE("poLineUpdateFailure", "One or more purchase order line record(s) cannot be updated"),
   VOUCHER_NOT_FOUND("voucherNotFound", "The voucher record is not found"),
   FUND_DISTRIBUTIONS_NOT_PRESENT("FundDistributionsNotPresent", "At least one fund distribution should present for every associated invoice line"),
-  FUND_DISTRIBUTIONS_PERCENTAGE_SUMMARY_MISMATCH("FundDistributionsPercentageSummaryMismatch", "Fund distributions percentage summary should be 100 for every associated invoice lines"),
+  FUND_DISTRIBUTIONS_SUMMARY_MISMATCH("FundDistributionsSummaryMismatch", "Fund distributions summary should be 100 % or equal to subtotal for every associated invoice lines"),
```

Comment:
Could you please use here lowerCamelCase in order to correspond all the other codes?

Reply:
done